README.md

```diff
@@ -5,259 +5,77 @@
+3. Create a new tmux session called ml-backend and label-studio in your EC2: `tmux new -s ml-backend` &  `tmux new -s label-studio`.
+4. Attach to the tmux session using `tmux a -t ml-backend` or `tmux a -t label-studio`. Navigate to the respective docker-compose and do `sudo docker compose build` to build all docker images.
+5. Then, inside the respective tmux session do `sudo docker compose up` and your services should be up and running.
+
+Tmux helps you keep processes alive even if you kill your shell instance. A cheatsheet regarding tmux commands can be found [here](https://tmuxcheatsheet.com).
+
+
+## 3. Setup label-studio
+1. Navigate to the URL of your EC2 (make sure you allow inbound traffic to the port on which label-studio listens in the security group of your EC2.)
+2. Create a new user, and log in using it.
+3. Create a new project from the UI.
+4. Go to settings --> Cloud Storage --> Add Source Storage.
+    Choose storage type `local files` and add the path to the data (the volume you have previously mounted)
+5. Set up the labeling interface: Go to Settings --> Labeling Interface --> Code and paste
+```html
+<View>
+  <Image name="image" value="$ocr"/>
+
+  <Labels name="label" toName="image">
+    <Label value="Material Description" background="green"/>
+    <Label value="Depth Interval" background="blue"/>
+  <Label value="Coordinates" background="#FFA39E"/></Labels>
+
+  <Rectangle name="bbox" toName="image" strokeWidth="3"/>
+  <Polygon name="poly" toName="image" strokeWidth="3"/>
+
+  <TextArea name="transcription" toName="image" editable="true" perRegion="true" required="true" maxSubmissions="1" rows="5" placeholder="Recognized Text" displayMode="region-list"/>
+</View>
 ```
-You can find all the generated assets, including SQLite3 database storage `label_studio.sqlite3` and uploaded files, in the `./mydata` directory.
-
-#### Override default Docker install
-You can override the default launch command by appending the new arguments:
-```bash
-docker run -it -p 8080:8080 -v $(pwd)/mydata:/label-studio/data heartexlabs/label-studio:latest label-studio --log-level DEBUG
-```
-
-#### Build a local image with Docker
-If you want to build a local image, run:
-```bash
-docker build -t heartexlabs/label-studio:latest .
-```
-
-### Run with Docker Compose
-Docker Compose script provides production-ready stack consisting of the following components:
-
-- Label Studio
-- [Nginx](https://www.nginx.com/) - proxy web server used to load various static data, including uploaded audio, images, etc.
-- [PostgreSQL](https://www.postgresql.org/) - production-ready database that replaces less performant SQLite3.
-
-To start using the app from `http://localhost` run this command:
+6. Add the ML Backends: Go to settings --> Model --> Add Model. The screenshots below outline the model settings. Make sure to adjust the IP of the Backend URLs to the private IP of your EC2 instance.
```

Comment:
Requiring to use the IP address here seems a little fishy to me.
Can't we configure the containers to use the same Docker network, and then simply use the service name instead of an IP address? (https://docs.docker.com/compose/networking/)